editor/src/components/canvas/canvas-strategies/strategies/set-padding-strategy.spec.browser2.tsx

```diff
@@ -21,7 +21,88 @@ import {
 } from '../../ui-jsx.test-utils'
 import { SetPaddingStrategyName } from './set-padding-strategy'
 
+const edgePieces: Array<EdgePiece> = ['top', 'bottom', 'left', 'right']
+
 describe('Padding resize strategy', () => {
+  it('Padding resize handle is not present for elements that have no padding set', async () => {
+    const editor = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`<div
+      data-testid='mydiv'
+      style={{
+        backgroundColor: '#0091FFAA',
+        position: 'absolute',
+        left: 28,
+        top: 28,
+        width: 612,
+        height: 461,
+      }}
+      data-uid='24a'
+    >
+      <div
+        style={{
+          backgroundColor: '#0091FFAA',
+          width: 22,
+          height: 22,
+        }}
+        data-uid='002'
+      />
+    </div>`),
+      'await-first-dom-report',
+    )
+
+    const canvasControlsLayer = editor.renderedDOM.getByTestId(CanvasControlsContainerID)
+    const div = editor.renderedDOM.getByTestId('mydiv')
+    const divBounds = div.getBoundingClientRect()
+    const divCorner = {
+      x: divBounds.x + 50,
+      y: divBounds.y + 40,
+    }
+
+    mouseClickAtPoint(canvasControlsLayer, divCorner, { modifiers: cmdModifier })
```

Comment:
as far as I understand, your only goal is to select the element, you can achieve that by dispatching 

```
await renderResult.dispatch([selectComponents([targetPath], false)], false)
```

this reduces the potential flakyness of the test, since now you are testing two things: click to select, _and_ padding controls

Reply:
I'd say keep this as is because the browser tests are closer to integration tests than unit tests, so we want this to be closer to real life use

Reply:
I just updated it 🤦‍♂️